one-pager-maker/tailwind.config.js

```diff
@@ -1,5 +1,7 @@
 /** @type {import('tailwindcss').Config} */
 export default {
+  mode: 'jit',
```

Comment:
Apparently `jit` enables classes such as `color-[#FFFFFF]`, but I think we can use these classes by default.

Reply:
I added it because it was written in https://v2.tailwindcss.com/docs/just-in-time-mode that it is very useful during development.  
Should I remove it?  

However, it seems that adding Purge settings is also necessary to make it work properly.  
What do you think?

Reply:
in the other project I could use just-in-time-mode so I thought that this might be unnecessary.
But if this is needed it is okay to leave it as it is.